addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -208,7 +209,7 @@ const Wysiwyg = Widget.extend({
             },
             commands: powerboxOptions.commands,
             categories: powerboxOptions.categories,
-            plugins: options.editorPlugins,
+            plugins: [...(options.editorPlugins || []), [MoveNodePlugin, { wysiwyg: this }]],
```

Comment:
This `wysiwyg` option doesn't seem to be used. Did I miss something ? Because if it's not used then we might remove support for plugin options... To be honest this was not even an option per se, it was basically misusing an options mechanism to pass some reference around... ^^'